sys/include/universal_address.h

```diff
@@ -28,7 +28,18 @@ extern "C" {
 #include <stdint.h>
 #include <stdlib.h>
 
-#define UNIVERSAL_ADDRESS_SIZE (16)         /**< size of the used addresses in bytes */
+/** @brief  size of the used addresses in bytes */
+/* determine the widest possible address type */
+#ifndef UNIVERSAL_ADDRESS_SIZE
+#define UNIVERSAL_ADDRESS_SIZE (0)       /* rather senseless default, should
```

Comment:
Hence, I think the solution is to let the unittest set this variable. Do you agree?